scripts/drop-downs.js

```diff
@@ -0,0 +1,11 @@
+import { getFacilitiesHTML } from "./facilities/facilities.js"
+import { getGovernorsHTML } from "./governors/governorComponent.js"
+
+export const getDropDownHTML = async () => {
+
+    const dropdownElement = document.querySelector(".dropdowns")
+    const governorsHTML = await getGovernorsHTML()
+    const facilitiesHTML = await getFacilitiesHTML()
+    dropdownElement.innerHTML = governorsHTML + facilitiesHTML  
```

Comment:
wrap governorsHTML and facilitiesHTML in a div with class dropdowns before adding to DOM